From 46b39b2b0a277008335cd5cf6731d88a8678b491 Mon Sep 17 00:00:00 2001
From: supechicken <me@supechicken666.dev>
Date: Fri, 21 Feb 2025 16:38:45 +0000
Subject: [PATCH] Fix Leanback Keyboard for ATV image

This patch remaps the Enter key to DPAD_CENTER for Leanback Keyboard to work

Change-Id: I201234d8a735c4eec5f6e0680c37bb5c13dcef21
Signed-off-by: supechicken <me@supechicken666.dev>
---
 data/keyboards/Generic.kl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/keyboards/Generic.kl b/data/keyboards/Generic.kl
index c81473ddcfc6..12193e0c0057 100644
--- a/data/keyboards/Generic.kl
+++ b/data/keyboards/Generic.kl
@@ -47,7 +47,7 @@ key 24    O
 key 25    P
 key 26    LEFT_BRACKET
 key 27    RIGHT_BRACKET
-key 28    ENTER
+key 28    DPAD_CENTER
 key 29    CTRL_LEFT
 key 30    A
 key 31    S
-- 
2.48.1
